docs(tem): concepts + menu and all files and folders (#940)
scaleway/docs-content: network/transactional-email/concepts.mdx

---
meta:
  title: Transactional Email - Concepts

>   description: This page explains all the concepts related to Scaleway’s Transactional Email

Reviewer: ```suggestion
  description: This page explains all the concepts related to Scaleway’s Transactional Email service 
```

wdyt ?

Author: lgtm! Je fix en local

</Concept>

<Concept opened>

## DKIM record

DKIM (DomainKeys Identified Mail) is an email security standard. It provides you with an encryption key and a digital signature that allows you to verify whether an email was altered. 

</Concept>

<Concept>

## DNS propagation

DNS propagation is the time frame it takes for DNS changes to be updated on the Internet.

</Concept>

<Concept>

##  IP reputation

IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, is assigned a good IP reputation. An IP address that sends high amounts of spam or viruses is assigned a bad IP reputation.IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.

</Concept>

<Concept>

##  MX record

A mail exchanger (MX) record is a DNS record that is necessary for delivering emails to your address. In simple DNS terms, an MX record is used to tell the world which mail servers accept incoming mail for your domain and where emails sent to your domain should be routed to.

</Concept>

<Concept>

##  Send API

> A Send API is the main API that is used to send transactional emails to recipients.

Reviewer: ```suggestion
A send API is the main API that is used to send transactional emails to recipients.
```

---
meta:
  title: Transactional Email - Concepts
  description: This page explains all the concepts related to Scaleway’s Transactional Email service
content:
  h1: Transactional Email - Concepts
  paragraph: This page explains all the concepts related to Scaleway’s Transactional Email service
tags: advanced settings
categories:
- network
---

Transactional Email is a platform that allows you to send personalized emails to individuals.

<Concept opened>

## Deliverability

Deliverability is the ability of an email to arrive in a recipient’s main email inbox.

</Concept>

<Concept opened>

## DKIM record

A DKIM (DomainKeys Identified Mail) record is an email security standard. It provides you with an encryption key and a digital signature that allows you to verify whether an email was altered.

> 

Reviewer 2: Remove line breaks?

</Concept>

<Concept>

##  MX record

A mail exchanger (MX) record is a DNS record that is necessary for delivering emails to your address. An MX record is used to tell the world which mail servers accept incoming mail for your domain and where emails sent to your domain should be routed to.

</Concept>

<Concept>

##  Send API

A send API is the main API that is used to send transactional emails to recipients.

</Concept>

<Concept>

##  SMTP relay

SMTP relay is the process of transferring an email from one mail server to another.

</Concept>

<Concept opened>

## SPF record

An SPF record is an email authentication method that allows you to protect senders and recipients from spam and phishing attacks. SPF also prevents spammers from sending emails on behalf of your domain. 

</Concept>

<Concept opened>

## Transactional Email

> Transactional Email is a platform that allows you to send transactional emails. Transactional emails are personalized emails that Scaleway sends to individuals in response to events they have triggered (e.g. password recovery, billing information, delivery updates, etc). 

Reviewer 2: ```suggestion
Transactional Email is a platform that allows you to send transactional emails. Transactional emails are personalized emails sent to individuals in response to events they have triggered (e.g. password recovery, billing information, delivery updates, etc). 
```

Reviewer 2: I made this suggestion as otherwise it sounds a bit like the emails are explicitly coming from Scaleway, rather than whatever domain you set up?
